src/lib/containers/markdown/MarkdownEditor.tsx

```diff
@@ -17,7 +17,85 @@ export const MarkdownEditor: React.FunctionComponent<MarkdownEditorProps> = ({
   const [currentTab, setCurrentTab] = useState<MarkdownEditorTabs>(
     MarkdownEditorTabs.WRITE,
   )
-  const [text, setText] = useState<string>()
+  const [text, setText] = useState<string>('')
+  const [selectionStart, setSelectionStart] = useState<number>(0)
+  const commandList = [
```

Comment:
Pull this constant out of the component, it doesn't need to be re-created every render